test/compress/collapse_vars.js

```diff
@@ -1047,10 +1046,9 @@ collapse_vars_object: {
     }
     expect: {
         function f0(x, y) {
-            var z = x + y;
             return {
                 get b() { return 7; },
-                r: z
+                r: x + y
```

Comment:
`AST_Accessor.has_side_effects()` on `get b() { return 7; }`, which aborts the scan previously.